packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -1,136 +1,88 @@
 import * as React from 'react';
+
```

Comment:
Will need to clean up the spacing in this file. Holding off for now because we still have a lot of work to do.